wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -382,6 +383,30 @@ public RepeatCommand repeatedly() {
     return new RepeatCommand(this);
   }
 
+  /**
+   * Decorates this command to run repeatedly, restarting until the command runs for the given times
+   * amount. The decorated command can still be canceled.
+   *
+   * <p>This is just syntactic sugar for {@code this.finallyDo(() ->
+   * counter[0]++).repeatedly().until(() -> counter[0] >= times)} which just means: runs the
+   * command, then increments the counter, repeatedly until the counter saturates.
+   *
+   * <p>Note: This decorator works by adding this command to a composition. The command the
+   * decorator was called on cannot be scheduled independently or be added to a different
+   * composition (namely, decorators), unless it is manually cleared from the list of composed
+   * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
+   * returned from this method can be further decorated without issue.
+   *
+   * @param times the count of times to run the command (inclusively).
+   * @return the decorated command
+   */
+  public ParallelRaceGroup repeatedly(int times) {
+    AtomicInteger counter = new AtomicInteger(0);
```

Comment:
I know we use this pattern in tests to get around the "effectively final" requirement in lambdas but I don't love using it in the actual "production" code- atomic types come with stronger guarantees than we need here (and thus increased runtime overhead).

Reply:
So, should I instead use an array with a size of 1?

Reply:
I think that would be fine